assets/webconfig/i18n/en.json

```diff
@@ -167,9 +184,9 @@
 	"conf_colors_color_intro" : "Create one or more calibration profiles, adjust each color, brightness, linearization and more.",
 	"conf_colors_smoothing_intro" : "Smoothing flattens color/brightness changes to reduce annoying distraction.",
 	"conf_colors_blackborder_intro" : "Skip black bars wherever they are. Each mode use another detection algorithm which is tuned for special situations. Higher the threshold if it doesn't work for you.",
+	"conf_network_net_intro" : "Network related settings which are applied to all network services.",
 	"conf_network_json_intro" : "The JSON-RPC-Port of this Hyperion instance, used for remote control.",
 	"conf_network_bobl_intro" : "Receiver for Boblight",
-	"conf_network_udpl_intro" : "Receiver for UDP",
 	"conf_network_fbs_intro" : "Google Flatbuffers Receiver. Used for fast image transmission.",
 	"conf_network_proto_intro" : "The PROTO-Port of this Hyperion instance, used for picture streams (HyperionScreenCap, Kodi Addon, ...)",
 	"conf_network_forw_intro" : "Forward all input to a second Hyperion instance which could be driven with another led controller",
```

Comment:
Should change wording here.
If multi instance is fully implemented: should we stick to forwarder?